src/rules/pyupgrade/rules/format_literals.rs

```diff
@@ -82,7 +93,9 @@ fn generate_call(
     let mut call = match_call(&mut expression)?;
 
     // Fix the call arguments.
-    call.args = generate_arguments(&call.args, correct_order)?;
+    if !is_sequential(correct_order) {
```

Comment:
@colin99d - Happy to follow-up and correct if I'm wrong, but I think it's sufficient to just do this check, and nothing else?